src/App.css

```diff
@@ -3,21 +3,41 @@
   padding: 0;
 }
 
-.App {
+.body {
   width: 100%;
   height: 100vh;
-  background-color: rgb(175, 172, 172);
+}
+
+.nav {
   display: flex;
-  justify-content: center;
-  align-content: center;
+  justify-content: space-between;
+  border: 2px solid rgb(72, 74, 70);
+  background-color: rgb(177, 173, 173);
+  height: 15vh;
+  width: 99.8%;
+  align-items: center;
 }
 
-.calc-container {
-  width: 500px;
-  height: 600px;
-  background-color: rgb(213, 210, 210);
-  border: 4px solid rgb(69, 68, 68);
-  margin-top: 100px;
+.navtitle {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 60px;
+  align-self: center;
+  margin-left: 5%;
+  color: rgb(53, 87, 117);
+}
+
+.navlist {
+  display: flex;
```

Comment:
[OPTIONAL] 
- To make your codebase DRY, you can make use of a `flex` class and just call it once in your CSS. Your are already having 3 `display: flex` under different classes.  Just imagine if your app has to grow. 

`.flex {
display: flex 
};`

With something like above, that would be the only `display:flex` in your CSS.